internal-import-file/import-file-misp/src/import-file-misp.py

```diff
@@ -2106,7 +2109,19 @@ def _update_container(self, bundle, entity_id):
                 if "x_opencti_id" in object
                 and object["x_opencti_id"] == container["id"]
             ][0]
-            container_stix["object_refs"] = [object["id"] for object in bundle]
+            if self._contains_container(bundle):
```

Comment:
I don't understand how this case could happen, since _update_container is only called when there is no container in the bundle : 

```
if self._contains_container(bundle):
    self.helper.log_info("Bundle contains container.")
else:
    self.helper.log_info(
        "No container in Stix file. Updating current container"
    )
    bundle = self._update_container(bundle, entity_id)
```

What did I miss ?